libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts

```diff
@@ -131,11 +131,13 @@ export class ExecuteBridgeRequest {
         afterResponse:
           command.afterResponse !== undefined ? [command.afterResponse] : [],
       },
-      /*
-       * Reject self-signed and invalid certificates in Production environments but allow them in Development
-       * as it's common for developers to use self-signed certificates in local environments.
-       */
-      rejectUnauthorized: environment.name.toLowerCase() === 'production',
+      https: {
+        /*
+         * Reject self-signed and invalid certificates in Production environments but allow them in Development
+         * as it's common for developers to use self-signed certificates in local environments.
+         */
+        rejectUnauthorized: environment.name.toLowerCase() === 'production',
+      },
```

Comment:
Fixes this error:
![image](https://github.com/user-attachments/assets/bbd2a965-806a-4919-b1b1-3722cffe07f0)

Reply:
Nice, I never saw that.